css-flexbox-exercise-2/style.css

```diff
@@ -0,0 +1,148 @@
+
+
+.flex-box-main-container {
+    display: flex;
+    flex-direction: column-reverse;
+    align-items: center;
+    background-color: rgb(158, 232, 250);
+    
+    
+    
+}
+
+.flex-box-main-container-item {
+    background-color: black;
+    display: flex;
+    flex-direction: column-reverse;
+    max-width: 500px;
+    width: 410px;
+    align-items: center;
+    border: 5px solid black;
+    border-top: 10px solid black;
+    border-radius: 5px;
+    
+}
+
+.nav-parent {
+    width: 390px;
+    max-width: 400px;
+}
+
+.nav-list {
+    list-style: none;
+    display: flex;
+    flex-direction: row;
+    background-color: rgb(238, 202, 42);
+    padding: 0;
+    margin: 5px 5px 40px 5px;
+    flex-basis: 100%;
+
+}
+
+.nav-list_item {
+    height: 100px;
+    width: 100px;
+    border: 2px solid transparent;
+    border-left: 1px solid black;
+    display: flex;
+    justify-content: center;
+    flex-direction: column;
+    align-items: center;
+    flex-basis: 100%;
```

Comment:
nie mozna mieszac flex-basis z width, to tak jakby podac width: 100px, a linijke nizej width: 100%